public/locales/en-US/grafana.json

```diff
@@ -880,9 +880,6 @@
       "subtitle": "Manage preferences across an organization",
       "title": "Default preferences"
     },
-    "performance-testing": {
```

Comment:
I don't know if this is done differently now, but these translations still exist on other languages (german, spanish, etc.). Do we need to remove them here?

Reply:
great question - i don't _think_ so, i think what happens is:
- the PR gets merged
- the new english file gets uploaded to crowdin via the new action
- the translations are updated (in this case to remove that string)
- the new translation files are raised as a PR
- we merge the PR, cycle complete

cc @joshhunt @L-M-K-B to help confirm that process 🙇

Reply:
ahh. interesting. @L-M-K-B I guess this is a side effect of i18n-extract no longer writing to the other language files.

I think the flow you pointed out seems correct. Lets see how it goes 👍